src/lib/serviceRoutines/getEntityTypes.cpp

```diff
@@ -48,6 +48,7 @@
 * URI parameters:
 *   - attributesFormat=object
 *   - collapse=true
+*   - options=noAttrDetail
```

Comment:
I understand this is the service routine for NGSIv1. In that case, `options=noAttrDetail` is not an option and its value is always "true" when invoking mongoAttributesForEntityType().

A comment shluld be added just above mongoAttributesForEntityType() to explain why is alwyas true, e.g. "Note that noAttrDetail is set to true in NGSIv1 operations, as NGSIv1 type browing doesn't actually provides attribute type detail in the API response" or something like that.